cmd/up/exec_windows.go

```diff
@@ -0,0 +1,40 @@
+//go:build windows
+// +build windows
+
+package up
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"syscall"
+)
+
+func (he *hybridExecutor) GetCommandToExec(ctx context.Context, cmd []string) (*exec.Cmd, error) {
+	var c *exec.Cmd
+	if runtime.GOOS != "windows" {
+		c = exec.Command("bash", "-c", strings.Join(cmd, " "))
+	} else {
+		binary, err := expandExecutableInCurrentDirectory(cmd[0], he.workdir)
+		if err != nil {
+			return nil, err
+		}
+		c = exec.Command(binary, cmd[1:]...)
+	}
+
+	c.Env = he.envs
+
+	c.Stdin = os.Stdin
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
+
+	c.Dir = he.workdir
+
+	c.SysProcAttr = &syscall.SysProcAttr{
```

Comment:
I am wondering if we couldn't make `c.SysProcAttr` conditional based on the OS, and remove the double file.

Reply:
i think this can be done at runtime but not on compilation. The error appearing was because when building for windows the compiler did not found this function for the platform...

Reply:
Not blocking comment, just a thought

Reply:
I see, thanks @teresaromero